From 2927120b539e0cfde16872b1b12dc37c9769432d Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Mon, 28 Dec 2015 15:17:06 +0200
Subject: [PATCH 0623/1240] a8k: mvebu: added board options documentaion for
 Armada-8k

- added list of board options for Armada-8k board, for fdt_config usage

Change-Id: I1936eb48978ed0b1c15c4fb2cfb94f5dbd955181
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26359
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
---
 doc/mvebu/a8k/board_options | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)
 create mode 100644 doc/mvebu/a8k/board_options

diff --git a/doc/mvebu/a8k/board_options b/doc/mvebu/a8k/board_options
new file mode 100644
index 0000000..a829577
--- /dev/null
+++ b/doc/mvebu/a8k/board_options
@@ -0,0 +1,32 @@
+Board options:
+--------------
+
+The below describes the board options for Armada-8k/7k
+For each board, the below defines the list of configurations that U-Boot support, using device tree blob.
+
+Option list of Armada-8040-RZ:
+
+option #	|	1	|	2	|	3	|	4	|	5	|	6	|	7	|
+---------------------------------------------------------------------------------------------------------------------------------
+SLM #		| SML1524	| SLM1521	| SLM1521	| SLM1521	| SLM1521	| SLM1522	| SLM1522	|
+---------------------------------------------------------------------------------------------------------------------------------
+SerDes:	lane0	| sgmii2	| pex_x4_l0	| pex_x4_l0	| pex_x4_l0	| pex_x4_l0	| sata1		| pex_x4_l0	|
+	lane1	| usb3h0	| pex_x4_l1	| sata0		| usb3h0	| usb3h0	| sata0		| sata0		|
+	lane2	| sgmii0	| pex_x4_l2	| kr		| kr		| sata0		| sgmii0	| sgmii0	|
+	lane3	| sata1		| pex_x4_l3	| sata1		| sata1		| sata1		| sgmii2	| sgmii2	|
+	lane4	| usb3h1	| pex_x1_0	| pex_x1_0	| usb3h1	| usb3h1	| rxauil0	| rxauil0	|
+	lane5	| pex_x1_1	| pex_x1_1	| pex_x1_1	| pex_x1_1	| pex_x1_1	| rxauil1	| rxauil1	|
+---------------------------------------------------------------------------------------------------------------------------------
+MPPs: 	SPI0	| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		|
+	SPI1	| [13-16]	| [13-16]	| [13-16]	| N/C		| [13-16]	| [13-16]	| [13-16]	|
+	NAND	| N/C		| N/C		| N/C		| [13,17-27]	| N/C		| N/C		| N/C		|
+	RGMII0	| N/C		| [0-12]	| [0-12]	| [0-12]	| [0-12]	| N/C		| N/C		|
+	RGMII1	| N/C		| [44-55]	| [44-55]	| [44-55]	| [44-55]	| N/C		| N/C		|
+	UART0	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	|
+	UART1	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	|
+	SD	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	|
+	TDM	| [0-11]	| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		|
+	TWSI	| [37-38]	| [37-38]	| [37-38]	| [37-38]	| [37-38]	| [37-38]	| [37-38]	|
+	SATA0	| [36]		| N/C		| [36]		| N/C		| [36]		| [36]		| [36]		|
+	SATA1	| [28]		| N/C		| [28]		| [28]		| [28]		| [28]		| N/C		|
+---------------------------------------------------------------------------------------------------------------------------------
-- 
1.9.1

